rx_signal_processing/dsp.hpp

```diff
@@ -40,25 +40,58 @@ std::vector<cudaDeviceProp> get_gpu_properties();
 void print_gpu_properties(std::vector<cudaDeviceProp> gpu_properties);
 
 
+typedef struct rx_slice
+{
+  double rx_freq;
+  uint32_t slice_id;
+  uint32_t num_ranges;
+  uint32_t beam_count;
+  float first_range; // km
+  float range_sep; // km
```

Comment:
Do both of these need to be floats, I'm wondering if sub-km range separations are possible? I guess if we have the ability to change the tx pulse length by 1us increments that makes total sense. [reader's thoughts, ignore.]